### DISCUSSION TITLE
关于让PCL支持安装Quilt的那些事

### DISCUSSION THREAD

**Discussion by Black-emmmm** [Meloong-Git/PCL#2763] [提问] [answered]
就无语，虽然PCL可以正常运行Quilt但是他识别成了Fabric
![image](https://github.com/Hex-Dragon/PCL2/assets/121375829/72f975c7-e648-4486-b689-1688aae777c9)
![image](https://github.com/Hex-Dragon/PCL2/assets/121375829/22cda0ef-fdcb-443d-a1e4-02661918eec3)

**Comment by WForst-Breeze** [comment, marked as answer]
PCL 尚不支持 Quilt 的识别和安装，等待第三方 API 适配后将会制作该功能
